src/Components/Footer.tsx

```diff
@@ -1,16 +1,120 @@
-import { CSSProperties } from "react";
+import {
+  faFacebook,
+  faInstagram,
+  faLinkedin
+} from "@fortawesome/free-brands-svg-icons";
+import { faEnvelope } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { useState } from "react";
+import { Link } from "react-router-dom";
+import styled from "styled-components";
 
 export function Footer() {
-  return(
-    <footer style={footerStyle}>
-      <h2>This is the footer!</h2>
-    </footer>
-  )
-}
+  const [buttonText, setButtonText] = useState("Gå med!");
+  const [buttonColor, setButtonColor] = useState("white");
+  const [newsLetterText, setNewsLetterText] = useState(
+    "Prenumerera på vårt nyhetsbrev!"
+  );
+  const [formSubmitted, setFormSubmitted] = useState(false);
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    setButtonText("✓");
+    setButtonColor("#76A887");
+    setNewsLetterText("Tack!");
+    setFormSubmitted(true);
+  };
+
+  return (
+    <MyFooter>
+      <Link to="*">
```

Comment:
Ska det vara länkar? de leder till 404 not found,De kanske inte behöver leda någonstans?